bin/setup.deps

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env ruby
+require 'pathname'
+require 'fileutils'
+
+# path to your application root.
+APP_ROOT = Pathname.new File.expand_path('../../',  __FILE__)
+
+
+def system!(*args)
+  system(*args) || abort("\n== Command #{args} failed ==")
+end
+
+def get_uid()
+  uid = `id -u`
+  uid.delete!("\n")
+end
+
+Dir.chdir APP_ROOT do
+  puts '== Installing application build deps =='
+  sudo_prefix = ''
+  sudo_prefix = "sudo" unless get_uid == '0'
+  system! "#{sudo_prefix} apt-get update && #{sudo_prefix} apt-get -y --no-install-recommends install libxml2 libxml2-dev postgresql-client postgresql-client-common libpq-dev"
+
+  puts "== Installing rbenv ruby manager to #{ENV['HOME']} =="
+  unless Dir.exist?("#{ENV['HOME']}/.rbenv/")
+    system! 'git clone https://github.com/sstephenson/rbenv.git $HOME/.rbenv'
```

Comment:
Perhaps `https://github.com/rbenv/rbenv.git` looks more official? This way you don't need to figure out who is that "sstephenson" every time you see this.